02-css/01-basic-layout/style.css

```diff
@@ -23,8 +23,10 @@ LAYOUT
 
 #wrapper {
   width: 960px;
-  margin: 0px auto;
   background: rgba(0,100,100,0.1);
+  padding: 5px 20px 20px 20px;
+  float: left;
+  margin-left: 10%;
```

Comment:
This looks pretty close, but positioning the core page content relative to either side of the page has some problems. It doesn't scale well on different browser sizes, won't accommodate a responsive design as well, and will never be perfectly centered. 

Try keeping `margin: 0px auto;`. You can find another way to get the wrapper to include the floats!